compiler/runtime.go

```diff
@@ -855,8 +856,16 @@ func (e *unaryEvaluator) EvalInt(scope Scope) (int64, error) {
 	if err != nil {
 		return 0, err
 	}
-	// There is only one integer unary operator
-	return -v, nil
+	var sign int64
+	switch e.op {
+	case ast.AdditionOperator:
+		sign = 1
+	case ast.SubtractionOperator:
+		sign = -1
```

Comment:
this was a bug, added a test case for that

Reply:
`+a` is valid according to the grammar?

Reply:
Ah yes it is. Thanks for this.